packages/contracts-bedrock/scripts/Deployer.sol

```diff
@@ -327,6 +327,18 @@ abstract contract Deployer is Script {
         _writeTemp(_name, _deployed);
     }
 
+    /// @notice Stubs a deployment retrieved through `get`.
+    /// @param _name The name of the deployment.
+    /// @param _addr The mock address of the deployment.
+    function prankDeployment(string memory _name, address _addr) public {
```

Comment:
`save` has similar functionality - is there a reason why you don't use `save`?

Reply:
Yes, it would write a temp file.